### PR TITLE
Add org logo to minimal program serializer

### PR DESCRIPTION
**Pull request by tuchfarber**
Similar to https://github.com/edx/course-discovery/pull/2844 we need
to store this value when we scan so we can use a cached version in
services like credentials.

### CODE REVIEW

**Review by staubina — APPROVED** [review]
Overall this PR makes sense to me. We should just make sure it does not bloat the number of SQL queries.

**Comment by tuchfarber** [comment]
Verified that due to existing prefetching we stay at 16 queries (checked via DDT).

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -361,9 +361,17 @@ def get_organization_logo_image_url(self, obj):
 
 
 class MinimalOrganizationSerializer(BaseModelSerializer):
+    certificate_logo_image_url = serializers.SerializerMethodField()
+
+    def get_certificate_logo_image_url(self, obj):
+        image = getattr(obj, 'certificate_logo_image', None)
+        if image:
+            return image.url
+        return None
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/api/serializers.py:370]: Nit: `return image and image.url`

**Review by Dillon-Dumesnil — APPROVED** [review]

File: course_discovery/apps/api/serializers.py (hunk 1 above)
Comment (tuchfarber) [reply course_discovery/apps/api/serializers.py:370]: Oh yeah, I just copied and pasted this from it's old location. I'll clean it up.

**Pull request merged**